.github/workflows/regression-report.yml

```diff
@@ -454,10 +454,30 @@ jobs:
 
       - name: Prepare environment
         run: |
-          mv .ci-temp/project.properties ./.ci-temp/contribution/checkstyle-tester/
-          mv .ci-temp/*.xml ./.ci-temp/contribution/checkstyle-tester/
+          mv .ci-temp/project.properties ./.ci-temp/test-configs/checkstyle-tester/
+          mv .ci-temp/*.xml ./.ci-temp/test-configs/checkstyle-tester/
```

Comment:
Please remind me why we need this files

Reply:
we have https://github.com/checkstyle/checkstyle/blob/f66c74cb841544b72719bc7cfaa33f7d82db7ab0/.github/workflows/regression-report.yml#L20

why we need "*.xml" files?

Reply:
in generate_report, we cd to checkstyle-tester and inside that we access the project.properties and config.xml. But, this is not related to default project properties per say. I mean say we have both the files, we move it to checkstyle-tester and execute the java jar from there.

Reply:
done